content/applications/finance/expenses.rst

```diff
@@ -477,37 +529,23 @@ and :guilabel:`Re-Invoicing Policy` by clicking the radio button next to the des
 - :guilabel:`At cost`: Expense product will invoice expenses at their real cost.
 - :guilabel:`At sales price`: Expense product will invoice the price set on the sale order.
 
-Create an expense
------------------
+Create an expense and expense report
+------------------------------------
 
 First, when :ref:`creating a new expense <expenses/new>`, the correct information needs to be
-entered in order to re-invoice a customer. Select the :guilabel:`Customer to Reinvoice` from the
-drop-down menu. Next, select the :guilabel:`Analytic Account` the expense will be posted to.
+entered in order to re-invoice a customer. Select the sales order the expense will appear on in the
+:guilabel:`Customer to Reinvoice` section, from the drop-down menu. Next, select the
+:guilabel:`Analytic Account` the expense will be posted to. After the expense(s) are created, the
```

Comment:
We introduce the instruction using expense (singular), so grammar should match accordingly.
```suggestion
:guilabel:`Analytic Account` the expense will be posted to. After the expense(s) is created, the
```